app/src/store/actions.ts

```diff
@@ -306,6 +306,10 @@ export async function searchBusiness ({ getters }, corpNum: string): Promise<Bus
   try {
     // first try to find business in Entities (Legal API)
     const data = await NamexServices.searchEntities(corpNum)
+    // Check if business is eligible for restoration NR by verifying restorationExpiryDate exists
+    if (getters.isRestoration && !data.restorationExpiryDate) {
+      throw new Error('This business is not eligible for restoration name request')
```

Comment:
Does this error get displayed anywhere? If not, would it possible to (easily/cleanly) show this error instead of "Business not found" under the lookup component?